cfgov/unprocessed/css/molecules/global-eyebrow.less

```diff
@@ -114,6 +114,20 @@
             display: block;
             padding-left: 0;
         }
+
+        .m-global-eyebrow_phone {
+          border-top: 1px solid @gray-40;
+          padding-top: unit( @grid_gutter-width / 2 / @base-font-size-px, rem );
+        }
+
+        .m-global-eyebrow_languages {
+          margin-bottom: 0;
+
+          .m-list_item {
+            margin-right: unit( @grid_gutter-width / @base-font-size-px, rem );
+            margin-bottom: unit( @grid_gutter-width / @base-font-size-px, rem );
```

Comment:
Does this actually increase the touch area? Wouldn't we have to add padding to the inner link elements `.m-list_item a` to expand the area that can be tapped on?

Reply:
Hmmmmmm good point. I'll attempt what you're saying.

Reply:
Updated in https://github.com/cfpb/cfgov-refresh/pull/5626/commits/a61f96972038e27bcd07fd55a45c477e7128f68c Also updated the phone number touch area.